File: Module3/case_module_3/case/src/main/java/models/person/Employee.java
package models.person;

import lombok.*;

import java.sql.Date;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class Employee {
    int id;
    String name;
    Date birthday;
    String id_card;
    double salary;
    String phone;
    String email;
    String address;
    String positions;
    String education_degree;
    String division;
}

File: Module5/GrandMercury/GrandMercuryProject/src/app/model/employee/Employee.ts
import {PositionEmpl} from "./PositionEmpl";
import {DivisionEmpl} from "./DivisionEmpl";
import {EducationEmpl} from "./EducationEmpl";
export interface Employee {
  id?: number;
  name?: string;
  dayOfBirth?: Date;

  id_card?: string;

  salary?: number;

  phone?: string;
  email?: string;
  address?: string;
  position?: PositionEmpl;

  division?: DivisionEmpl;

  education?: EducationEmpl;
  status?: number
}
